corehq/apps/app_manager/templates/app_manager/partials/case_list_session_endpoint.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
This is pretty similar to `session_endpoints.html` - did you consider DRYing them up? If you don't combine them into a single template, you might also consider putting them in the same file, just so someone editing one part is more likely to see the other part and consider if it should also be edited.

Reply:
> did you consider DRYing them up? I

Is there an existing example I can look at to see how can I do this better ? I think there are several settings that are similar on HQ like `lable --- free input` but I was not able to find a code instance where we are doing something else than defining the UI components again.

Reply:
I meant to update `session_endpoints.html` to use variables so it would work for both the existing endpoint input and the new one you're adding. Similar to [nav_menu_media_single_type.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media_single_type.html).

Although it looks like there are 4 things to replace (label, help, input id, value), which is kind of a lot for a fairly short template, so it's debatable how much of an improvement it'd be.

Reply:
Got it, I went for merging the two templates in the same file instead - https://github.com/dimagi/commcare-hq/pull/33183/commits/cbf504df046d8e74f38087077767cf900228ad59

Reply:
nice 👍